howtos/how_to_request_cached_data.py

```diff
@@ -0,0 +1,139 @@
+            PubSubPlusClientException: If the operation could not be performed
+            IllegalStateException: If the service is not running
+        """
+        topic = TopicSubscription.of(subscription_expression)
+        subscription_to_receive_cached_followed_by_live_messages = CachedMessageSubscriptionRequest.cached_first(cache_name,
+                                                                                                                 topic,
+                                                                                                                 cache_access_timeout,
+                                                                                                                 max_cached_messages,
+                                                                                                                 cached_message_age)
+        correlation_id = 12345
+        completion_listener = MyCorrelationCompletionListener()
+
+        direct_receiver.requestCached(subscription_receive_cached_followed_by_live_messages, correlation_id, completion_listener)
+
+    def create_subscription_to_receiver_only_cached_messages(direct_message_receiver: 'DirectMessageReceiver',
+                                                             subscription_expression: str,
+                                                             cache_name: str,
+                                                             cache_access_timeout):
+        """
+        An example how to create and perform CachedMessageSuibscriptionRequest to receive cached messages
+        only, when available; no live messages are expected to be received.
+        Args:
+            direct_message_receiver(DirectMessageReceiver): An already configured and connected receiver.
+            subscription_expression(str): topic subscription expression
+            cache_name(str): name of the solace cache to retrieve from
+            cache_access_timeout(int): solace cache request timeout, in milliseconds
+        Raises:
+            InterruptedException: If any thread has interrupted the current thread
+            PubSubPlusClientException: If the operation could not be performed
+            IllegalStateException: If the service is not running
+        """
+        subscription_request_to_receive_cached_followed_by_live_messages = CachedMessageSubscriptionRequest.cached_only(cache_name,
+                                                                                                                        TopicSubscription.of(subscription_expression),
+                                                                                                                        cache_access_timeout)
+        correlation_id = 12345
+        completion_listener = MyCorrelationCompletionListener()
+
+        receiver.requestCached(subscription_request_to_receive_cached_followed_by_live_messages, correlation_id, completion_listener)
```

Comment:
Variable is named `direct_message_receiver` not `receiver`
Method is named `request_cached`

Reply:
Please review fix

Reply:
Missed re-naming the parameter as well (Variable is named `direct_message_receiver` not `receiver`)

Reply:
Please review fix.

Reply:
LGTM